release-team/role-handbooks/branch-manager/README.md

```diff
@@ -1,235 +1,307 @@
-* `./gcbmgr stage master --build-at-head`
-   * Rather than having GCB find a point in the git history that had clean CI signal and building automatically from that point, we instead indicate we want to build explicitly from the git head.
-   * This takes some time (approximately 1 hour is the current norm).  It’s building all the bits for a bunch of target operating systems and hardware architectures.
-* `./gcbmgr tail`
-   * You can watch the build progress.
-   * Scan the output for failures and warnings.
-   * For a successful build information is emitted on your next steps, including a build version identifier for the staged build, for example “v1.12.0-alpha.0.2622+7ac32a4f7a7ecf”.  This string is passed on to the release scripting, which is nicely emitted for you ready to copy and paste.
-   * TODO: what guidance can we give on typical failures and triaging?
-
-## Alpha Mock Release
+
+Early in the release cycle, it is likely that the build might fail. By default the `gcbmgr stage master` command automatically looks for a place where [release master blocking tests](https://k8s-testgrid.appspot.com/sig-release-master-blocking) have green results, which traditionally has not happened in Kubernetes on an ongoing basis.
+
+WE REALLY WANT (and need) TO GET THERE. Quality needs to be a continual focus. But in the meantime, acknowledging today especially for an early alpha or beta release, it is possible to just build via:
+
+`./gcbmgr stage master --build-at-head`
+   * Rather than having `gcbmgr` pick a candidate by analyzing test data from the commit history that had no fails and building automatically from that point, we instead indicate we want to build explicitly from HEAD (the last commit on the current branch).
+   * This takes time (approximately 1 hour is the current norm).  It’s building all the bits for a bunch of target operating systems and hardware architectures.
+
+`./gcbmgr list`
+   * You should now see your new job running.
+
+`./gcbmgr tail`
+   * To observe the output log for the build (same as on Google Cloud Console).
+   * Scan output for failures and warnings.
+   * Successful build information is emitted on your next steps, including a build version identifier for the staged build, for example “v1.12.0-alpha.0.2622+7ac32a4f7a7ecf”. This string is passed on to the release scripting, which is nicely emitted for you ready to copy and paste.
+   * **NB**: what guidance can we give on typical failures and for debugging?
```

Comment:
Lets shift this TODO/NB to an issue to improve in the future.  Perhaps in dialog with shadows during the next cycle we see some commonly asked questions which we can provide up front in the document.

Reply:
Continued in #696